packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -260,6 +260,7 @@ check_docker() {
 check_docker
 
 # start buildkite-agent
+systemctl set-environment "BUILDKITE_TERMINATE_INSTANCE_AFTER_JOB=${BUILDKITE_TERMINATE_INSTANCE_AFTER_JOB}"
```

Comment:
This is going to make it into the job's env yeah? It seems like a useful thing for a job to know, so I'm not opposed to exposing it. But typically, env variables set by the elastic stack are disclosed in an environment hook that sources `/var/lib/buildkite-agent/cfn-env`.

Indeed, that file is the way for configuration set by template parameters to effect the behaviour of various scripts that run on the elastic stack.

You're using a different way of doing that here, which I generally think is better. But it only works for scripts run through systemd. 🤷‍♂️

I'll approve it if it's just addded to `cfn-env` though. That file is generated in https://github.com/buildkite/elastic-ci-stack-for-aws/blob/ee348e4976c9eb4696dc44c3b4f08e59f073059f/packer/linux/conf/bin/bk-install-elastic-stack.sh#L99-L100